### PR TITLE
Ce 3110

### PR DESCRIPTION
**Pull request by kamilkoterba**
https://wikia-inc.atlassian.net/browse/CE-3110

Added shortcut key hint and welcome hint

@Wikia/community-engineering

### CODE REVIEW

File: extensions/wikia/TemplateClassification/TemplateClassification.i18n.php [hunk 1]
```diff
@@ -39,6 +39,7 @@
 	'template-classification-edit-modal-error' => 'Unfortunately we were not able to save this template type. Would you mind trying again?',
 	'template-classification-indicator' => 'Template type:',
 	'template-classification-notification-error-retry' => 'Something went wrong and we could not save the template type. Could you retry it using the link under the page title?',
+	'template-classification-open-modal-key-tip' => 'Open classification dialog (Ctrl+Shift+K)',
```
Comment (lukaszkonieczny) [inline extensions/wikia/TemplateClassification/TemplateClassification.i18n.php]: This combination can will vary on different OS. I'm wondering shouldn't we choose just one letter as `accesskey` and then key combination is handle by OS and browser (http://www.w3schools.com/tags/att_global_accesskey.asp)

File: extensions/wikia/TemplateClassification/styles/TemplateClassification.scss [hunk 2]
```diff
@@ -1,7 +1,14 @@
-$tc-error-bg-color: #e1390b;
-$tc-modal-text-color: #656e78;
-$tc-name-color-checked: #1a5eb8;
-$tc-name-color: #39424d;
+@import "skins/shared/color";
+
+$tc-name-color-checked: $color-links;
+
+@if $is-dark-wiki {
+	$tc-name-color: lighten(mix($color-text, $color-links, 75%), 5%);
```
Comment (lukaszkonieczny) [inline extensions/wikia/TemplateClassification/styles/TemplateClassification.scss:6]: Did you consider using existing (calculated) wiki colors?

Comment (lukaszkonieczny) [inline extensions/wikia/TemplateClassification/styles/TemplateClassification.scss:6]: Discussed offline :+1:

File: extensions/wikia/TemplateClassification/TemplateClassificationController.class.php [hunk 3]
```diff
@@ -49,4 +49,16 @@ public function getTemplateClassificationEditForm() {
...
 	}
+
+	/**
+	 * Sets user property confirming that user has seen hint on TemplateClassification edit entry point,
+	 * so it won't show up again
+	 */
+	public function dismissWelcomeHint() {
+		$user = $this->getContext()->getUser();
+		if ( !$user->isAnon() ) {
+			$user->setGlobalPreference(\Wikia\TemplateClassification\View::HAS_SEEN_HINT, true);
```
Comment (adamkarminski) [inline extensions/wikia/TemplateClassification/TemplateClassificationController.class.php]: Use `use Wikia\TemplateClassification\View` and also - spaces `(  )` :)

File: extensions/wikia/TemplateClassification/TemplateClassification.hooks.php [hunk 4]
```diff
@@ -223,4 +226,18 @@ private function getTemplateTypeForEdit( \Title $title, $wikiaId ) {
 
 		return $templateType;
 	}
+
+	private function addWelcomeHintAssets( \OutputPage $out, \User $user ) {
+		global $wgCityId;
+		if ( !$user->getGlobalPreference( \Wikia\TemplateClassification\View::HAS_SEEN_HINT ) ) {
```
Comment (adamkarminski) [inline extensions/wikia/TemplateClassification/TemplateClassification.hooks.php]: The `View` class is in the same namespace.

**Comment by adamkarminski** [comment]
LGTM, @azarek - ready for re-testing

**Comment by azarek** [comment]
LGTM :)

**Pull request merged**